help/configure-home-view.md

```diff
@@ -46,6 +46,13 @@ organization settings:
    shortcut twice to exit the settings and navigate to your home view
    (<kbd>Ctrl</kbd> + <kbd>[</kbd> or <kbd>Esc</kbd> if enabled).
 
+!!! tip ""
+
+    You can also hover over **Inbox**, **Recent conversations**, or
+    **All messages** in the left sidebar, click on the **ellipsis**
+    (<i class="zulip-icon zulip-icon-more-vertical"></i>), and
+    select the option to make that your home view.
```

Comment:
Maybe something like this instead?

    You can also hover over your desired home view in the left sidebar, and choose the option to **make it your home view** in the **ellipsis**
    (<i class="zulip-icon zulip-icon-more-vertical"></i>) menu.

I'm not sure it's ideal, but I found "select the option to make that your home view" a bit hard to parse.